docs/Server.md

```diff
@@ -134,10 +134,20 @@ interface by having the following methods: `info`, `error`, `debug`, `fatal`, `w
       return child;
     },
   };
-  
+
   const fastify = require('fastify')({logger: customLogger});
   ```
 
+<a name="factory-disable-request-logging"></a>
+### `disableRequestLogging`
+By default, when logging is enabled, Fastify will issue an `info` level log
+message when a request is received and when the response for that request has
+been sent. By setting this option to `true`, these log messages will be disabled.
+This allows for more flexible request start and end logging by attaching
+custom `onRequest` and `onResponse` hooks.
+
++ Default: `false`
```

Comment:
Can you add an example on how to write a hook to do this logging?

Reply:
Done.